### PR TITLE
Nice pdf ticket

### PR DESCRIPTION
**Pull request by IliaStepanov**
Nicer pdf file and mailbox with app name.

### CODE REVIEW

**Comment by IliaStepanov** [comment]
[[Killing feature] make nice PDF ticket](https://trello.com/c/7DMFJfzk/72-killing-feature-make-nice-pdf-ticket)

File: src/main/java/com/epam/lowcost/services/implementations/PDFService.java [hunk 1]
```diff
@@ -2,64 +2,63 @@
...
-        document.add(new Paragraph(String.format("Boarding Pass # %d", ticket.getId())).setFont(font));
-// Create a List
-        List baseInfo = new List()
-                .setSymbolIndent(12)
-                .setListSymbol("\u2022")
-                .setFont(font);
+        Document document = new Document();
+        PdfWriter writer = PdfWriter.getInstance(document, new FileOutputStream(String.format("Ticket_№%d.pdf",ticket.getId())));
+        document.open();
+        Image img = Image.getInstance("ticket_template.jpg");
```
Comment (EugeneGrim) [inline src/main/java/com/epam/lowcost/services/implementations/PDFService.java]: Move the file from project's root directory to any other folder (may be resources..)

Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/services/implementations/PDFService.java]: Made a card for that on trello.

**Comment by IliaStepanov** [comment]
[[BE] create normal root for pdf files](https://trello.com/c/q92Wdoj3/114-be-create-normal-root-for-pdf-files)

File: src/main/java/com/epam/lowcost/services/implementations/PDFService.java (hunk 1 above)
Comment (IliaStepanov) [reply src/main/java/com/epam/lowcost/services/implementations/PDFService.java]: Also moved template image from root to resources folder.

**Pull request merged**